src/app/core/theme.component.ts

```diff
@@ -1,8 +1,44 @@
 import { Component } from '@angular/core';
+import { EventCard, EventCardComponent } from '../features/event';
 
 @Component({
   selector: 'app-theme',
+  imports: [EventCardComponent],
   standalone: true,
-  template: ` <h1>Storybook-like route</h1> `,
+  template: `
+    <h1>Storybook-like route</h1>
+    <h2>Event Card Component</h2>
+    <div style="padding: 8px 8px;">
+      <app-event-card [eventCard]="event"></app-event-card>
+    </div>
+  `,
 })
-export default class ThemeComponent {}
+export default class ThemeComponent {
+  event: EventCard = {
+    id: 1,
+    eventInfo: {
+      name: 'Giga Koks Turbo Event',
+      category: {
+        id: 1,
+        name: 'Piwo',
+      },
+      startDate: '20.01.2023',
+      hashtags: [
+        {
+          id: 1,
+          value: 'piwo',
+        },
+        {
+          id: 1,
```

Comment:
id się powtarza to samo, niby nic takiego, ale może spowodować potencjaaaaalnie problem potem

Reply:
Rozumiem obawy ale są to tylko i wyłącznie mockowane dane do testu, bez wpływu na napisany komponent. Poprawione.